src/app/Shared/Dialog.component.tsx

```diff
@@ -10,11 +12,19 @@ export function DialogComponent(props: {
     dialogTitle: string;
     dialogContentText: JSX.Element;
     dialogContent: JSX.Element;
-    dialogButtons: JSX.Element;
+    cancelButton: JSX.Element | string;
+    submitButton: JSX.Element | string;

```

Comment:
Good workaround. But the header export functionality should still be moved to a container in later step, so that the export is triggered with the onSubmit event of the dialog button not the click event of the link that is contained within the submitButton input component.

Reply:
I know. That's not part of this ticket.